public/app/percona/ui-events/UIEvents.service.ts

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable @typescript-eslint/no-explicit-any,@typescript-eslint/consistent-type-assertions, */
+
+import { logger } from '@percona/platform-core';
+
+import { ApiRequest } from 'app/percona/shared/helpers/api';
+import { DashboardUsageEvent } from 'app/percona/ui-events/events/dashboard';
+import { FetchingEvent } from 'app/percona/ui-events/events/fetching';
+import { NotificationErrorEvent } from 'app/percona/ui-events/events/notification';
+
+const api = new ApiRequest({ baseURL: '/v1/ui-events' });
+
+interface UIEventsStoreRequest {
+  notifications: NotificationErrorEvent[];
+  fetching: FetchingEvent[];
+  dashboard_usage: DashboardUsageEvent[];
+}
+
+interface TelemetryStoreResponse {}
+
+export const UIEventsService = {
+  async store(body: UIEventsStoreRequest): Promise<void> {
+    try {
+      await api.post<TelemetryStoreResponse, any>('/Store', body, true);
```

Comment:
if we don't have a response schema, just use `object`, no need for an empty interface